cfme/common/provider.py

```diff
@@ -700,12 +700,21 @@ def validate_stats(self, ui=False):
         self.mgmt.disconnect()
 
     @variable(alias='rest')
-    def refresh_provider_relationships(self, from_list_view=False):
+    def refresh_provider_relationships(self, from_list_view=False,
+                                       wait=0, delay=1, refresh_delta=10):
+        """
+        wait[seconds], 0 for no wait
+        """
+        if not wait and (delay != 1 or refresh_delta != 10):
```

Comment:
you don't need this if statement since those parameters are used only in wait_for. those won't be passed to wait_for if wait == 0

Reply:
'those won't be passed to wait_for if wait == 0' -> that's the reason why I put that logger.info on the next line. If someone calls refresh_provider_relationship(delay=60), I think there should be at least that logger.info, something is wrong. But I have no problem with removing it if you find unnecessary.

Reply:
okay, you have `if wait` below. you can just add else to it with logger.info